api.go

```diff
@@ -211,11 +211,13 @@ type (
 		Pull       *StreamPull `json:"pull,omitempty"`
 		PullRegion string      `json:"pullRegion,omitempty"`
 		CreatorID  *CreatorID  `json:"creatorID,omitempty"`
+		ProjectID  string      `json:"projectID,omitempty"`
```

Comment:
Is this coming from Studio? Shouldn't this and the creator be without the capitalized D?

Reply:
Good point